<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userPurchaseFood</class>
 <widget class="QDialog" name="userPurchaseFood">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Heading">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="foodItemLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="foodItemEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="foodItemToPurchaseEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="foodItemToPurchaseLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchFoodButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
